onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -505,8 +505,8 @@ std::unique_ptr<IExecutionProvider> CreateExecutionProviderInstance(
             return tensorrt_provider_factory->CreateProvider();
           }
         }
-        LOGS_DEFAULT(WARNING) << "Failed to register " << type << ". Please reference https://onnxruntime.ai/docs/execution-providers/TensorRT-ExecutionProvider.html#requirements to ensure all dependencies are met.";
     }
+    LOGS_DEFAULT(WARNING) << "Failed to register " << type << ". Please reference https://onnxruntime.ai/docs/execution-providers/TensorRT-ExecutionProvider.html#requirements to ensure all dependencies are met.";
```

Comment:
We're not registering anything here. We're only creating an instance of the provider. So, the error msg seems a bit misleading, no?
Besides is this really a failure? ORT_TENSORRT_UNAVAILABLE might be non-empty if the user is on a CPU machine.

Reply:
it is in the registration path.

Reply:
We can change the verb register to create to make it more consistent with where it's being called from.
CreateExecutionProviderInstance() is called by RegisterExecutionProviders()